server/scsynth/SC_GraphDef.cpp

```diff
@@ -604,10 +604,10 @@ SCErr GraphDef_DeleteMsg(World *inWorld, GraphDef *inDef)
 	packet.addtag('s');
 	packet.adds((char*)inDef->mNodeDef.mName);
 
-	ReplyAddress *users = inWorld->hw->mUsers;
-	int numUsers = inWorld->hw->mNumUsers;
-	for (int i=0; i<numUsers; ++i) {
-		SCErr err = SendReplyCmd_d_removed(inWorld, packet.size(), packet.data(), users+i);
+	Clients::iterator it;
+	for (it = inWorld->hw->mUsers->begin(); it != inWorld->hw->mUsers->end(); ++it){
```

Comment:
Hi guys. @brianlheim I'm fine with all the C++ comments above. Sorry, my C++ is a little old school! :-) I've got several deadlines at the moment, so am a bit swamped, but if someone else wants to make those changes and test I'm happy with that.